shell/src/main/java/org/apache/accumulo/shell/commands/SetIterCommand.java

```diff
@@ -363,23 +363,54 @@ public String description() {
     return "sets a table-specific or namespace-specific iterator";
   }
 
-  @Override
-  public Options getOptions() {
+  // Set all options common to both iterators and shell iterators
+  public void setBaseOptions(Options options) {
```

Comment:
I this refactoring, very nice. Could make this method protected.

Reply:
Updated access from public to protected.